app/src/main/res/layout/item_bill.xml

```diff
@@ -37,65 +37,91 @@
 
             <TextView
                 android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
+                android:layout_height="match_parent"
                 android:background="@color/colorPrimary"
+                android:gravity="center"
+                android:minWidth="@dimen/dp_20"
                 android:padding="@dimen/dp_2"
                 android:text="@{position + 1 + ``}"
                 android:textColor="@color/color_white"
                 android:textSize="@dimen/sp_18"/>
 
             <LinearLayout
                 android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_marginEnd="@dimen/dp_10"
-                android:layout_marginRight="@dimen/dp_10"
+                android:layout_height="match_parent"
                 android:layout_weight="1"
-                android:background="@color/color_white"
-                android:orientation="horizontal"
-                android:paddingLeft="@dimen/dp_5"
-                android:paddingRight="@dimen/dp_5"
-                android:paddingStart="@dimen/dp_5">
+                android:orientation="vertical"
+                android:paddingLeft="@dimen/dp_3"
+                android:paddingStart="@dimen/dp_3">
 
                 <TextView
                     android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
+                    android:layout_height="0dp"
+                    android:layout_margin="@dimen/dp_1"
                     android:layout_marginEnd="@dimen/dp_30"
                     android:layout_marginRight="@dimen/dp_30"
+                    android:layout_weight="1"
```

Comment:
WARNING: [NestedWeights] Nested weights are bad for performance
 Layout weights require a widget to be measured twice. When a LinearLayout with non-zero weights is nested inside another LinearLayout with non-zero weights, then the number of measurements increase exponentially.